academy/3-ibc/3-channels.md

```diff
@@ -67,6 +74,8 @@ You can find the implementation of `ChannelOpenInit` in the the [`msg_server.go`
 
 The important part to note in this code snippet is that an application module has capabilities for the requested port. Therefore, an application module can only use a channel and port if the application owns the capability for that port and the module which attempting to open a channel is the module we have granted capabilities to in `app.go`:
 
+<!-- TODO: add more content on capabilites based on Colin's deep dive -->
```

Comment:
Is this work being tracked in an issue? If so maybe we can link the issue in the comment.